radixdlt-engine/src/main/java/com/radixdlt/engine/RadixEngine.java

```diff
@@ -131,7 +131,27 @@
   private REConstructor actionConstructors;
   private ConstraintMachine constraintMachine;
 
+  private boolean isShutdown;
+
   // Used only in tests
```

Comment:
I would like to propose that we make sure this kind of thing won't happen in the babylon repo.

Reply:
you mean the comment or something else?